src/api/api.cc

```diff
@@ -7797,11 +7803,12 @@ i::Handle<i::JSArray> SetAsArray(i::Isolate* i_isolate, i::Object table_obj,
 }
 }  // namespace
 
-Local<Array> Set::AsArray() const {
+Local<Array> Set::AsArray(const v8::KeyIterationParams*) const {
   i::Handle<i::JSSet> obj = Utils::OpenHandle(this);
   i::Isolate* i_isolate = obj->GetIsolate();
   API_RCS_SCOPE(i_isolate, Set, AsArray);
   ENTER_V8_NO_SCRIPT_NO_EXCEPTION(i_isolate);
+  // TODO
```

Comment:
???

Reply:
I forgot to fix up `Set`. Fixed now. Livetests running now.